hyperwallet/api.py

```diff
@@ -1450,6 +1452,212 @@ def deactivatePaperCheck(self,
 
     '''
 
+    Transfers
+    https://portal.hyperwallet.com/docs/api/v3/resources/transfers
+
+    '''
+
+    def createTransfer(self,
+                         data=None):
```

Comment:
I cloned this project and after I execute this command **Make test** then I faced some code style warning like this: _hyperwallet/api.py:1461:26: E127 continuation line over-indented for visual indent ... _

Reply:
Fixed